lib/polyfill/mediasource.js

```diff
@@ -39,7 +39,7 @@ shaka.polyfill.MediaSource = class {
       shaka.polyfill.MediaSource.patchCastIsTypeSupported_();
     } else if (safariVersion) {
       // TS content is broken on Safari in general.
-      // See https://github.com/google/shaka-player/issues/743
+      // See https://github.com/shaka-project/shaka-player/issues/743
       // and https://bugs.webkit.org/show_bug.cgi?id=165342
```

Comment:
Oh hey, this issue was updated a few weeks ago!
They say it's fixed now, and I just ran the sample content in Safari and it seems to be fixed?
So maybe we can remove this. As another CL, obviously.

Reply:
Sounds good.  We should do a round of TODO cleanup before we push out v4.